### ISSUE TITLE
[BUG] - Datepicker lijkt niet WCAG compliant

### ISSUE THREAD

**Issue by driesva** [milieuinfo/webcomponent-vl-ui-datepicker#55]
**Omschrijf het probleem**
Bij de a11y tests krijgen we:

```
Accessibility Violations
---
Buttons must have discernible text
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"
  Element has no title attribute or the title attribute is empty
<button id="button" is="vl-button-input-addon" type="button" class="js-vl-datepicker-toggle vl-input-addon">
---
Form elements must have labels
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty
<input id="input" is="vl-input-field" block="" type="text" class="js-vl-datepicker-input vl-input-field--block flatpickr-input vl-input-field">
---
```

**Hoe te reproduceren**
Stappen om het probleem te reproduceren en indien mogelijks een code snippet:

`<vl-datepicker format="d/m/Y"></vl-datepicker>`

**Gewenst gedrag**
WCAG compliancy

**Omgeving (gelieve aan te vullen):**
 -  Frontend unit tests

**Comment by coemans** [comment]
Staat genoteerd. Ik wacht nog op feedback van EAG om WCAG werk te kunnen plannen. Het is de bedoeling dat elke webcomponent voorzien wordt van geautomatiseerde a11y testen. Uiteraard mogen gekende problemen al gemeld worden, maar ze zullen binnenkort sowieso automatisch naar boven komen.

**Comment by coemans** [comment]
Ondertussen werden al de webcomponenten gecontroleerd op WCAG richtlijnen. De nodige verbeteringen zullen hier nog uitgevoerd worden, maar ticket wordt afgesloten vanwege duplicate intern.
